sphinx/tutorial/rose/furthertopics/rose-arch.rst

```diff
@@ -0,0 +1,281 @@
+   rename-parser=^.*list(?P<tag>.*)$
+   source=file_list?
+
+
+Description
+-----------
+
+You have now created a suite that defines three tasks:
+
+``make_files``
+   Sets up the files and ``ARCHIVING/`` directory for ``archive_files_rsync/``
+   and ``archive_files_scp/`` to "archive", move, date to.
+``archive_files_rsync``
+   "Archives", rsync's files to the ``ARCHIVING/`` folder in the
+   ``$ROSE_DATA/`` directory.
+``archive_files_scp``
+   "Archives", scp's the renamed files and moves them to the ``ARCHIVING/``
+   folder in the ``$ROSE_DATA/`` directory.
+
+Save your changes and run the suite::
+
+   rose suite-run
+
+View the suite output using ``rose suite-log`` and inspect the output of the
+``make_files``, ``archive_files_rsync`` and ``archive_files_scp`` tasks.
+
+
+Results Of "Archiving"
+----------------------
+
+Change to the ``$ROSE_DATA/ARCHVING/`` directory of the suite i.e::
+
+   cd ~cylc-run/SUITE_ID/share/data/ARCHIVING/
+
+List the directory by typing::
+
+   ls
+
+You should see the following returned:
+
+.. code-block:: none
+
+   dir  file_zipped.tar  files  rename  solo.file
+
+Change directory to ``files/`` and list the files::
+
+   cd files
+   ls
+
+The following should be returned:
+
+.. code-block:: none
+
+   file_list1  file_list3
+
+Change directory to ``ARCHIVING/dir/`` and list the files::
+
+   cd ..
+   cd dir
+   ls
+
+The following should be returned:
+
+.. code-block:: none
+
+   file_list1  file_list2 file_list3
+
+.. note::
+
+   These were all of the files in the ``$ROSE_DATA/`` directory.
+
+Change diectory to ``ARCHIVING/rename/`` and list the files::
+
+   cd ..
+   cd rename
+   ls
+
+The following should be returned:
+
+.. code-block:: none
+
+   1_1_file_list1 1_2_file_list2 1_3_file_list3 
+
+These are the renamed files.
+
+Most users will have there own system or location that they wish to archive
+their data to. Here the example shown uses ``rsync`` and ``scp``. Further
+information on Linux commands ``rsync``, ``scp`` Please refer your own site
+specific archiving solutions and seek site specfic advice.
+
+
+Arch Settings
+-------------
+
+Some settings that can be used are described below.
+
+.. TODO - link to reference material
+
+   Also see: Built-in Applications: rose_arch
+
+Above ``.tar`` was used to compress the file. However, ``compress=gzip``
+can also be used. Note either of these commands can be used to compress a
+file or a folder/directory.
+
+In the above example a regular expression (reg exp) was used by the
+``rename-parser``, for example, ``^.*list(?P<tag>.*)$``, where:
+
+* ``^`` = startof a string.
+* ``$`` = end of a string.
+* ``.`` = any character.
+* ``*`` = greedy (all).
+* ``?P<NAME>`` = named group.
+
+.. note::
+
+   .. _python flavor: https://docs.python.org/2/howto/regex.html
+
+   ``rose arch`` uses the `python flavor`_ for regular expressions.
+
+In the above example source was used to accept a list of glob patterns.
```

Comment:
``source`` (monospace font) or else this does not make much sense. Moreover 'glob' strikes me as too specialised given the non-computing background of much our audience.